addons/account/models/account_tax_report.py

```diff
@@ -394,3 +394,15 @@ def always_carry_over_and_set_to_0(self, options, line_amount, carried_over_amou
         # Lines below 0 will be set to 0 and reduce the balance of the carryover.
         # Lines above 0 will be set to 0 and increase the balance of the carryover.
         return (0, 0)
+
+    def _get_carryover_destination_line(self, options):
+        """
+        Return the destination lines for the carryover for this tax report line.
+        It could return two or more lines in some particular configuration.
+
+        E.g. Italian tax report, two company in the same tax unit but with different fiscal year end date.
```

Comment:
It doesn't make sense anymore, right ?

We're always returning a single line now, aren't we ?